apps/docs/.vitepress/plugins/utils/extractImports.test.ts

```diff
@@ -0,0 +1,282 @@
+import { ErrorCode, FuelError } from '@fuel-ts/errors';
+
+import * as extractImportsMod from './extractImports';
+import fs from 'fs';
+import { expectToThrowFuelError } from '@fuel-ts/errors/test-utils';
+
+const {
+  collectImportStatements,
+  combineImportStatements,
+  parseIgnoreImportFlags,
+  validateImports,
+  extractImports,
+} = extractImportsMod;
+
+describe('extractImports', () => {
+  describe('parseIgnoreImportFlags', () => {
+    it('should return an empty set for no ignore flags', () => {
+      const lines = ['import { something } from "somewhere";', 'const x = 5;'];
+      expect(parseIgnoreImportFlags(lines)).toEqual(new Set());
+    });
+
+    it('should correctly parse a single ignore flag', () => {
+      const lines = ['// #ignore { testImport }'];
```

Comment:
I couldn't find any snipped using this `#ignore` variation.

I suggest adding clear docs around `#import` and `#ignore` on the PR description.

Reply:
@arboleya I've removed the `#ignore` import functionality since it was not being used.